src/wp-admin/css/about.css

```diff
@@ -732,6 +732,15 @@
 	}
 }
 
+@media screen and (max-width: 1200px) {
```

Comment:
@umeshsinghin, thanks for the review.

I'm not just trying to target mobile devices. It's for both mobile and tablet devices. Also, while practically testing for responsiveness, the value 1200px is found to suit the needs the best, otherwise, the UI just breaks without specifically styling it for the tablet devices.

Reply:
@yogeshbhutkar, According to the standard layout, a 1200px container width is typically for desktop view, which may cause conflicts on several devices. If targeting iPads, consider using a max-width: 960px, as this effectively covers all iPad devices